### DISCUSSION TITLE
I want to receive my GitHub Benefits

### DISCUSSION THREAD

**Discussion by YfelipeG** [community/community#50720] [GitHub Education]
Hi!

Currently I've tried to activate the GitHub Student Developer Pack, **but** I've received various emails that say my university (Católica del Norte Fundación Universitaria) is not in my country (Colombia). I've multiple times but all of them have failed.

These are the current messages I receive in my email:

> The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
> 
> The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.
> 
> Your document does not appear to include a date demonstrating current academic affiliation. For countries utilizing non-standard calendars, you may need to capture the original document beside one with a converted date. You may include multiple documents in your image, so long as they are legible.
> 
> You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

What I can do solve this issue?

Could you help me?

Thanks in advance!

**Comment by JOAQUINPABLO001** [comment]
+51929172527

**Comment by samus-aran** [comment]
Hi @YfelipeG 👋 

We're happy you're here. How did you get on? Did you do anything differently in your next submission?
